site/docs/getting-started/hello-world.md

```diff
@@ -283,3 +283,30 @@ w4.trace("Hello world!");
 :::tip
 Press `F8` to toggle the WASM-4 developer tools, which displays system state, memory usage, and more.
 :::
+
+## Start Function
```

Comment:
I like surfacing the start function, but do you think a code sample is necessary? Maybe this could be a short `::: note` above (where we explain `update()`) that introduces `start()` and what it can be used for.

Reply:
Changed it to a note, like you suggested. I think it will work well.